src/test/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfigTests.java

```diff
@@ -0,0 +1,41 @@
+package io.jenkins.plugins.artifact_manager_s3;
+
+import java.util.logging.Logger;
+import io.jenkins.plugins.artifact_manager_jclouds.JCloudsArtifactManagerFactory;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import jenkins.model.ArtifactManagerConfiguration;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class S3BlobStoreConfigTests {
+
+    private static final Logger LOGGER = Logger.getLogger(JCloudsVirtualFileTest.class.getName());
+
+    public static final String CONTAINER_NAME = "container-name";
+    public static final String CONTAINER_PREFIX = "container-prefix";
+    public static final String CONTAINER_REGION = "us-west-1";
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void checkConfigurationManually() {
```

Comment:
OK…not very interesting though. If you are going to bother writing a test, use `JenkinsRule.configRoundtrip` to catch actual mistakes. Which I think would fail with the PR as written, due to the missing `@DataBoundSetter`s.